Test.C

```diff
@@ -282,15 +327,18 @@ int main(){
         int equipSellAmount;
         int equipSellTransaction;
         char equipSellAnswer;
-        
+
+        printf("\n");
         printf("|----------------------------- Equipment Selling ------------------------------|\n");
         printf("Equipment that you own:\n");
+        printf(" # | Equipment Name                       | Cost  | Message Amount | Inventory |\n");
+        printf("---|--------------------------------------|-------|----------------|-----------|\n");
         for(int i=0; i<NUM_EQUIPMENT; i++){
-            printf("%s %s %d %d\n", equipmentStock[i].name, equipmentStock[i].description, equipmentStock[i].cost, equipmentStock[i].messageAmount);
-            printf("Amount of %s that is in your inventory: %d\n", equipmentStock[i].name, player.equipmentInventory[i]);
+            printf(" %d | %38s | %5d | %5d\n", i + 1, equipmentStock[i].name, equipmentStock[i].cost, equipmentStock[i].messageAmount, player.equipmentInventory[i]);
```

Comment:
Missed one of the parameters you listed.
```suggestion
            printf(" %d | %36s | %5d | %14d | %9d |\n", i + 1, equipmentStock[i].name, equipmentStock[i].cost, equipmentStock[i].messageAmount, player.equipmentInventory[i]);
```